### PR TITLE
Add Inazuma Eleven Go Galaxy 3: Big Bang and Supernova

### PR DESCRIPTION
**Pull request by EternelBlademaster**
Adds Inazuma Eleven Go Galaxy: Bigbang and Supernova.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/inazuma-eleven-go-galaxy-bigbang-and-supernova/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Inazuma Eleven Go Galaxy: Bigbang and Supernova"
```
Comment (Leo626) [inline games/inazuma-eleven-go-galaxy-bigbang-and-supernova/game.dat]: `Bigbang` should be `Big Bang`.

File: games/inazuma-eleven-go-galaxy-bigbang-and-supernova/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description = "Inazuma Eleven GO 3: Galaxy is a role-playing video game and sports video game for the Nintendo 3DS developed and published by Level-5. It's the third game of the Inazuma Eleven GO series and the sixth installment of the Inazuma Eleven games. There are two versions of the game, Big Bang and Supernova."
```
Comment (Leo626) [inline games/inazuma-eleven-go-galaxy-bigbang-and-supernova/game.dat]: * You can drop the `3:` in the game's title in order to keep it consistent.
* `role-playing video game and sports video game` could be shortened to `role-playing and sports video game`.
* `for the Nintendo 3DS` could be dropped.

**Comment by Leo626** [comment]
@EternelBlademaster The other version should be added like this.

```
[[releases]]
title = "000400000010BB00"
region = "jpn"
release_date = "2013-12-05"

[[releases]]
title = "000400000010BA00"
region = "jpn"
release_date = "2013-12-05"
```
This is causing Travis CI to fail.

**Comment by CodingKoopa** [comment]
That icon has your mouse in it. It has to be exactly captured, without anything like that in it.

**Comment by EternelBlademaster** [comment]
The boxart hasn't white border, I checked.

**Comment by CodingKoopa** [comment]
It's still low quality though, [this](https://gamefaqs.akamaized.net/box/9/4/0/293940_front.jpg) one is better.

**Comment by EternelBlademaster** [comment]
Hi, I made all the necessary changes, I hope everything is settled now.

**Comment by Leo626** [comment]
One small thing to address. The title of the directory is `inazuma-eleven-go-galaxy-bigbang-and-supernova`. It should match the `title` but with the spaces replaced with hypens - `inazuma-eleven-go-galaxy-3-big-bang-and-supernova`.

**Pull request merged**